api/src/main/java/net/kyori/adventure/text/TranslatableComponent.java

```diff
@@ -27,8 +27,21 @@
 import org.checkerframework.checker.nullness.qual.NonNull;
 
 /**
- * A translatable component.
+ * A component that can display translated text.
```

Comment:
I'm assuming that this is server side translated text?

Reply:
Or client. Adventure has a "server-side translation" library (`translation` package), but the `TranslatableComponent` is not limited to them.